deps/source_package/update

```diff
@@ -1,13 +0,0 @@
-#! /bin/sh
-
-if ! which dch >/dev/null; then
-	echo >&2 "please install devscripts"
-	exit 1
-fi
-
-if ! which debootstrap >/dev/null; then
-	echo >&2 "please install debootstrap"
```

Comment:
you can use `for` loop here instead of using 2 `if` statements.

Reply:
This file has been removed. Since it is optional.